template/.github/workflows/build.yml.j2

```diff
@@ -376,6 +376,8 @@ jobs:
           wget https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases/download/1.9.1/preflight-linux-amd64
           chmod +x preflight-linux-amd64
       - name: Check container
-        run: ./preflight-linux-amd64 check container "$IMAGE_TAG" > preflight.out
+        run: |
+          ARCH_FOR_PREFLIGHT="$(arch | sed -e 's#x86_64#amd64#' | sed -e 's#aarch64#arm64#')"
+          ./preflight-linux-amd64 check container "$IMAGE_TAG" --platform "${ARCH_FOR_PREFLIGHT}" > preflight.out
```

Comment:
The `amd64` suffix in  `/preflight-linux-amd64` seems weird, but when it works I'm happy ^^

Reply:
Thanks for approving! Yes, preflight seems to supply the prefix itself. If we pass `linux/amd64` it fails with:
`preflight .../stackable-experimental/hive:3.1.3-stackable24.3.0 --platform linux/amd64 --> no child with platform linux/linux/amd64`